stdlib/public/runtime/ImageInspectionELF.cpp

```diff
@@ -98,7 +99,11 @@ void swift_addNewDSOImage(const void *addr) {
   const swift::MetadataSections *sections =
       static_cast<const swift::MetadataSections *>(addr);
 
-  record(sections);
+  // We cast off the const in order to update the linked list
+  // data structure. This is safe to do since we don't touch 
+  // any other fields.
```

Comment:
`clang-format`?  Seems overly narrow in width.

Reply:
`clang-format` left it as is!